### DISCUSSION TITLE
Juiste manier vermelden data uit software van externe parijen

### DISCUSSION THREAD

**Discussion by tin900** [ed2c/landelijke_data#3] [Announcements]
Juiste manier van vermelden van data uit software die instellingen bij externe partijen afsluiten. Denk hierbij aan Canvas LMS van Instructure en TermTime van Semestry.

## Externe Software Systemen
Deze categorie omvat data afkomstig uit softwaresystemen die onderwijsinstellingen afnemen van externe leveranciers. Deze systemen bieden vaak continue datastromen die toegankelijk zijn via API's of grafische interfaces

| Leverancier | Doel | Documentatie | CEDA repository | Publieke informatieproducten |
|--------|--------|--------|--------|--------|
| Canvas LMS - Instructure | Leer management systeem | [API](https://canvas.instructure.com/doc/api/all_resources.html) | [vvcanvas](https://github.com/vusaverse/vvcanvas) | - |
| TermTime - Semestry | Roostering | - |  [vvtermtime](https://github.com/vusaverse/vvtermtime) | - |

**Comment by CorneeldH** [comment]
Ha Tomer, interessant. Dit is nu een iets andere structuur. Je kunt leverancier splitsen in leverancier (Instructure, Semestry) en bron (Canvas, Termtime).  Daarbij is het ipv per 'bestand' nu per API, dat lijkt me okay. Er mist een frequentie, die heb je vaak niet bij API natuurlijk, dan kun je gewoon real-time zetten.

**Reply by tin900** [reply]
Ben jij het hier mee eens?

| Leverancier | Bron | Onderwijstype | Doel | Documentatie URL | CEDA repository | Frequentie | Publieke informatieproducten |
|-------------|------|---------------|------|-----------------|-----------------|-------------|----------------------------|
| Instructure | Canvas LMS | MBO, HBO, WO | Leer management systeem | [API documentatie](https://canvas.instructure.com/doc/api/all_resources.html) | - | Real-Time | [R package vvcanvas](https://github.com/vusaverse/vvcanvas) |
| Semestry | Termtime | MBO, HBO, WO | Roostering |  | - | Real-Time | [R package vvtermtime](https://github.com/vusaverse/vvtermtime) |

**Comment by tin900** [comment]
Kunnen wij de packages die zijn gelinkt wellicht plaatsen onder `Publieke informatieproducten`?

**Reply by CorneeldH** [reply]
meh, het zijn dan weer niet Publieke informatieproducten. Wellicht er gewoon bij zetten: "Externe software bronnen met Vusaverse packages" en dan ipv CEDA repository VusaVerse repository? Iets in die trant. Dan zetten we het onderaan gewoon als extra service.

**Reply by tin900** [reply]
Beperken wij ons niet op die manier?

Stel collega's aan een andere instelling hebben een package in bijvoorbeeld python voor een bepaald prodcut dan valt dat niet onder vusaverse natuurlijk.

Voor mijn begrip, de definitie voor publiek dataproduct is dashboards?

**Comment by CorneeldH** [comment]
Ja het is beperking, maar ook legends of the game zoals Martine hadden nog geen package gemaakt voor CEDA. Bram had er eentje. Degene die package oid heeft weet ons echt wel te vinden en dan kunnen we het gemakkelijk aanpassen.

Kleinere stukjes code kunnen we ook ceda repo zetten (zie mails).

Publiek informatieproduct kan dashboard zijn tabel of pdf of PowerPoint met cijfers en of plaatjes. Zolang het publiek is en informatie bevat over onderwijs. Dus packages en pipelines niet,dat is code om tot zoiets te komen :-)

**Reply by tin900** [reply]
Helder. Eens!

Commit 5d64840f6fcedaac7f5af07b89a2d65f2f2eb3e7 introduceert de header en de informatie (tot nu).
